.gitlab-ci.yml

```diff
@@ -12,6 +12,7 @@ variables:
   GIT_SUBMODULE_STRATEGY: recursive
   CCACHE_DIR: /cache
   CCACHE_MAXSIZE: 100G
+  CCACHE_CPP2: "yes"
```

Comment:
Do we even need this anymore with 
https://github.com/espressomd/docker/pull/35?

Reply:
no